### ISSUE TITLE
Question about CIOU loss 

### ISSUE THREAD

**Issue by HaolyShiit** [AlexeyAB/darknet#6056]
Hello, I have a question.

The loss type is " ciou " in yolov4.cfg. Why not calculate loss using " tot_ciou_loss " instead of " tot_iou_loss  "?

https://github.com/AlexeyAB/darknet/blob/51d59cf30ad357aa1c831700e2ea36ad2f025183/src/yolo_layer.c#L624-L640

like this:

```
 if (l.iou_loss == GIOU) {
            avg_iou_loss = count > 0 ? l.iou_normalizer * (tot_giou_loss / count) : 0;
        }
        else if (l.iou_loss == CIOU) {
            avg_iou_loss = count > 0 ? l.iou_normalizer * (tot_ciou_loss / count) : 0;
        }
        else {
            avg_iou_loss = count > 0 ? l.iou_normalizer * (tot_iou_loss / count) : 0;
        }
```
Thank u !
